pro/src/pages/Onboarding/OnboardingOfferIndividual/OnboardingOfferIndividual.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { FormLayout } from 'components/FormLayout/FormLayout'
+import editFullIcon from 'icons/full-edit.svg'
+import connectStrokeIcon from 'icons/stroke-connect.svg'
+import { RadioButtonWithImage } from 'ui-kit/RadioButtonWithImage/RadioButtonWithImage'
+
+import { ActionBar } from '../components/ActionBar/ActionBar'
+import { OnboardingLayout } from '../components/OnboardingLayout/OnboardingLayout'
+
+import styles from './OnboardingOfferIndividual.module.scss'
+
+const ONBOARDING_OFFER_PROCEDURE = {
+  MANUAL: 'MANUAL',
+  AUTOMATIC: 'AUTOMATIC',
+} as const
+// eslint-disable-next-line no-redeclare, @typescript-eslint/naming-convention
+type ONBOARDING_OFFER_PROCEDURE =
+  (typeof ONBOARDING_OFFER_PROCEDURE)[keyof typeof ONBOARDING_OFFER_PROCEDURE]
```

Comment:
Comme c'est très local comme type, pourquoi ne pas partir sur un truc du genre
```suggestion
 type ONBOARDING_OFFER_PROCEDURE = 'manual' | 'automatic'
```

Reply:
En effet, je pensais que potentiellement on pourrait avoir besoin de ce type dans d'autres étapes de l'onboarding, mais ça reste assez local comme tu dis. Je vais changer ça …